backend/src/envPlatform/envservice.bal

```diff
@@ -0,0 +1,157 @@
+import ballerina/http;
+import ballerina/openapi;
+
+listener http:Listener ep0 = new(9090);
+
+@openapi:ServiceInfo {
+    contract: "resources/openapi_v3.yaml"
+}
+@http:ServiceConfig {
+    basePath: "/"
+}
+
+service envService on ep0 {
+
+    @http:ResourceConfig {
+        methods:["GET"],
+        path:"/applications"
+    }
+    resource function resource_get_applications (http:Caller caller, http:Request req) returns error? {
+
+    }
+
+    @http:ResourceConfig {
+        methods:["POST"],
+        path:"/applications",
+        body:"body"
+    }
+    resource function resource_post_applications (http:Caller caller, http:Request req,  TreeRemovalForm  body) returns error? {
+
+    }
+
+    @http:ResourceConfig {
+        methods:["GET"],
+        path:"/applications/{application-id}"
+    }
+    resource function resource_get_applications_application-id (http:Caller caller, http:Request req,  string &#x27;application\-id) returns error? {
```

Comment:
We can't have a method name which contains a dash, this will give out a compilation error. So we can't have `resource_get_applications_application-id` we should either have `resource_get_applications_application_id` or `resource_get_applications_applicationId`. This applies to all the other places in this file. @yashodgayashan pls check on this. Lemme know how you generated these files.

Reply:
getApplicationsByID should be a suitable name. I just generated the code using ballerina openapi gen-service

Reply:
Will either have to change the yaml file or the code. I'm not sure modifying the code is the best way. @sanjulamadurapperuma your point of view?

Reply:
We can change **application-id** to **applicationId** in the yaml, for example, to avoid this in the generation

Reply:
I will Change those. But we cannot do it for service method names.

Reply:
Yeah what I'm asking is the best method. Modifying the code or the yaml file? For me it's modifying the yaml file.

Reply:
@deshankoswatte the issue is we have to change the both in this case as mentioned by @sanjulamadurapperuma we can change path parameters in the yaml file but for the method names we have to edit it by our own.

Reply:
Yeah, yaml is the best method we should try, but in cases where conflicts still cannot be resolved by editing the yaml, then we will have to modify the code.

In that situation, I don't think modifying the generated code will cause significant issues because they are not being regenerated every time we build unless otherwise, we run the gen-service.